### PR TITLE
[WIP]Add OnMarkerCreated,OnMarkerDeletting,etc callbacks

### PR DESCRIPTION
**Pull request by amay077**
#29 

Add listed callback methods in android side ``MapRenderer.cs`` that provides chance of customize marker.

* ``OnMarkerCreating``
* ``OnMarkerCreated``
* ``OnMarkerDeleting``
* ``OnMarkerDeleted``

### CODE REVIEW

File: Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ public MapRenderer() : base()
                 new PolylineLogic(),
                 new PolygonLogic(),
                 new CircleLogic(),
-                new PinLogic(),
+                new PinLogic(OnMarkerCreating, OnMarkerCreated, OnMarkerDeleting, OnMarkerDeleted),
```
Comment (milena23) [inline Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs]: It will be greate to Add same functionality for iOS as well.

Comment (amay077) [reply Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs]: Sure, I will add to iOS side too! 👍

**Comment by milena23** [comment]
👍

**Comment by amay077** [comment]
Hi @milena23 ,
I released Xamarin.Forms.GoogleMaps v2.3.0-beta1 that includes this PR.

https://www.nuget.org/packages/Xamarin.Forms.GoogleMaps/2.3.0-beta1

**Pull request merged**